packages/search/src/routes/root.ts

```diff
@@ -118,28 +118,28 @@ const root: FastifyPluginAsync = async (_fastify, _options): Promise<void> => {
       },
     } as const,
     handler: async function (request, reply) {
-      const { approach } = request.body;
-      const askApproach = fastify.approaches.ask[approach];
+      const { approach } = request.body.context ?? {};
+      const askApproach = fastify.approaches.ask[approach ?? 'rtr'];
       if (!askApproach) {
         return reply.badRequest(`Ask approach "${approach}" is unknown or not implemented.`);
       }
 
-      const { question, overrides, stream } = request.body;
+      const { question, context, stream } = request.body;
       try {
         if (stream) {
           const buffer = new Readable();
           // Dummy implementation needed
           buffer._read = () => {};
           reply.type('application/x-ndjson').send(buffer);
 
-          const chunks = await askApproach.runWithStreaming(question, overrides ?? {});
+          const chunks = await askApproach.runWithStreaming(question, (context as any) ?? {});
           for await (const chunk of chunks) {
```

Comment:
as unknown maybe?

Reply:
We could type it better with context can be any object, I'd say `any` is better suited here as unknown isn't assignable to anything but itself